src/renderer/components/SystemStats.tsx

```diff
@@ -28,27 +28,38 @@ export const SystemStats = memo(() => {
     return (
         <HStack>
             {data &&
-                data.map((usage) => (
-                    <Tooltip
-                        borderRadius={8}
-                        key={usage.label}
-                        label={`${usage.percent.toFixed(1)}%`}
-                        px={2}
-                        py={1}
-                    >
-                        <Box key={usage.label}>
-                            <CircularProgress
-                                capIsRound
-                                color={usage.percent < 90 ? 'blue.400' : 'red.400'}
-                                size="42px"
-                                trackColor="var(--bg-700)"
-                                value={usage.percent}
-                            >
-                                <CircularProgressLabel>{usage.label}</CircularProgressLabel>
-                            </CircularProgress>
-                        </Box>
-                    </Tooltip>
-                ))}
+                data.map((usage) => {
+                    let color;
+                    if (usage.percent < 50) {
```

Comment:
Isn't 50% a bit low for what is pretty much a warning color? E.g. my RAM typically hovers around 50%, but I still have 16GB of RAM left, so there's no issue at all. 

Maybe we could raise it to 75%? That feels more like something that would warrant getting the user's attention.

Reply:
No problem, I will raise it to 75%.